iwallet/account.go

```diff
@@ -50,11 +50,11 @@ type accounts struct {
 	Account []*acc
 }
 
-// accountCmd represents the account command
+// accountCmd represents the account command.
 var accountCmd = &cobra.Command{
 	Use:   "account",
-	Short: "KeyPair manage",
-	Long:  `Manage account of local storage`,
```

Comment:
What does 'of local storage' mean actually?

Reply:
this linux account. 
accounts inside "~/.iwallet"